RELEASE_NOTES

```diff
@@ -1,6 +1,42 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.9.0 (21 August 2025)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Note! Though it is a minor release it includes breaking changes of the user
+interface for triggering build jobs. For details see below and documentation at
+https://www.eessi.io/docs/bot/#building
+
+Bug fixes:
+* revised and updated app.cfg.example and README.md (#325)
+* do not overwrite config value for scontrol command (#335)
+
+Improvements:
+* add support for cloning target repository via ssh (#300)
+* major refactoring of the definition of build targets and breaking change of
+  the user interface to trigger build (#312, #329, #331, #337)
+  * `arch_target_map` is replaced by `node_type_map` which provides a more
```

Comment:
point to section in README for more details?

Can use link https://github.com/EESSI/eessi-bot-software-layer?tab=readme-ov-file#architecturetargets-section